{"uuid": "9ecf5c30-0903-499f-9e73-73cbcf8abc33", "children": ["970f2fce-f44a-4e84-b466-3649862f3cc8", "a58d97a4-6fa4-4423-8c05-b43329f25d03", "b5abe6a6-7e46-43d1-8475-ca5d06e046ef", "9d0d4227-d23e-4d9d-930d-e66db9f85302", "ad0fbdc5-b8e3-4706-89c8-9584e8cf4fc4", "542a410a-f6a6-47bd-b18e-51d63cbc558e", "9dfdc976-411e-482c-bea8-ff8809fd5966", "bfd03156-27cd-446a-95d8-4e786562b383", "c0f0f2f4-83c7-41ad-bc75-68c64bb072ae", "da578c80-362e-4cdd-9245-26188386950d", "ce749d06-cdc8-4347-ab16-a4fc19c56aa6", "ddb4b0d9-57c0-4052-9c25-4620ded75454", "694ebc4d-dffd-482d-a9a1-a36f322c9fe4", "862d6572-41ba-4c98-920e-4219cbe50018"], "befores": [{"name": "test_bucketcode", "status": "broken", "statusDetails": {"message": "KeyError: 'data'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 924, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/apple/PycharmProjects/zhongshujiaoben/Test_case/Test_APIcase/test_bucket.py\", line 19, in test_bucketcode\n    bucketcode_zcjk=bz.bucket_chuanjian()\n  File \"/Users/apple/PycharmProjects/zhongshujiaoben/Customer_platform/API/bucket/bucket_zhuce.py\", line 45, in bucket_chuanjian\n    bucketcode=res.json()[\"data\"]\n"}, "start": 1621995771078, "stop": 1621995771214}], "start": 1621995771078, "stop": 1621995771479}